benchmark/config.py

```diff
@@ -100,6 +119,25 @@ def __init__(self, runtime_config=None):
                 # Extract relevant settings from config file
                 if "enabled" in runtime_config.vcf_to_zarr:
                     self.enabled = config_str_to_bool(runtime_config.vcf_to_zarr["enabled"])
+                if "alt_number" in runtime_config.vcf_to_zarr:
+                    alt_number_str = runtime_config.vcf_to_zarr["alt_number"]
+
+                    if str(alt_number_str).lower() in ["none", "auto"]:
+                        self.alt_number = None
+                    elif isint(alt_number_str):
+                        self.alt_number = int(alt_number_str)
```

Comment:
Very minor point as I expect most users will know what they're doing, but you could add an else branch raising an error, to catch the case where an invalid value is provided. Also applies to chunk_length and chunk_width below.